src/bundle/Resources/public/scss/_buttons.scss

```diff
@@ -35,3 +35,8 @@
         margin-right: 0;
     }
 }
+
+.btn.disabled, 
+.btn:disabled {
```

Comment:
Maybe it should be a generic selector like: `.disabled, *:disabled`

Reply:
Reviewed and updated.